<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistWindow</class>
 <widget class="QMainWindow" name="PlaylistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: rgb(52,51,64);
}
QLabel {
    color: rgb(255,255,255);
}
QPushButton {
    border: none;
    border-radius: 15px;
    min-width: 60px;
    max-width: 60px;
    min-height: 60px;
    max-height: 60px;
    background-color: rgb(255,255,255);
    color: rgb(52,51,64);
    padding: 0px;
}
QSlider::groove:horizontal {
    height: 10px;
    background: red;
    border-radius: 5px;
}
QSlider::handle:horizontal {
    background: rgb(255,255,255);
    width: 20px;
    height: 20px;
    margin: -7px -7px;
    border-radius: 10px;
}
QTreeView {
    background-color: rgb(40,40,50);
    color: rgb(255,255,255);
    border: 2px solid rgb(255,255,255);
    border-radius: 5px;
}
QTreeView::item:selected {
    background-color: rgb(70,70,80);
    color: rgb(255,255,255);
}
QHeaderView::section {
    background-color: rgb(60,60,70);
    color: rgb(255,255,255);
    border: 1px solid rgb(80,80,90);
    padding: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="infoWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <family>Ubuntu Sans</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    border: 2px solid rgb(255,255,255);
    border-radius: 5px;
    padding: 5px;
}</string>
         </property>
         <property name="text">
          <string>No file playing</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="artistLabel">
         <property name="font">
          <font>
           <family>Ubuntu Sans</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    padding: 5px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="albumLabel">
         <property name="font">
          <font>
           <family>Ubuntu Sans</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    padding: 5px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="timeLayout">
         <item>
          <widget class="QLabel" name="currentTimeLabel">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Sans</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="positionSlider">
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal
{
	height : 10px;
	width : 800px;
	background : red;
	border-radius : 5px;
}
QSlider::handle:horizontal
{
	background : rgb(255,255,255);
	width : 10px;
	height : 20px;
	margin : -7px -7px;
	border-radius : 10px;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalTimeLabel">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Sans</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="controlsWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="previousButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D7;
    border: none;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #1E88E5;
}

QPushButton:pressed {
    background-color: #005A9E;
}</string>
         </property>
         <property name="icon">
          <iconset theme="media-skip-backward"/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D7;
    border: none;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #1E88E5;
}

QPushButton:pressed {
    background-color: #005A9E;
}</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start"/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D7;
    border: none;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #1E88E5;
}

QPushButton:pressed {
    background-color: #005A9E;
}</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-stop"/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D7;
    border: none;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #1E88E5;
}

QPushButton:pressed {
    background-color: #005A9E;
}</string>
         </property>
         <property name="icon">
          <iconset theme="media-skip-forward"/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="volumeButton">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D7;
    border: none;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #1E88E5;
}

QPushButton:pressed {
    background-color: #005A9E;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>149</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal
{
	height : 5px;
	width : 120px;
	background : red;
	border-radius : 2px;
}
QSlider::handle:horizontal
{
	background : rgb(255,255,255);
	width : 10px;
	height : 20px;
	margin : -5px -5px;
	border-radius : 5px;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="playlistView">
      <property name="styleSheet">
       <string notr="true">QTreeView {
    background-color: rgb(40,40,50);
    color: rgb(255,255,255);
    border: 2px solid rgb(255,255,255);
    border-radius: 5px;
}

QTreeView::item:selected {
    background-color: rgb(70,70,80);
    color: rgb(255,255,255);
}

QTreeView::item:hover {
    background-color: rgb(60,60,70);
}

QHeaderView::section {
    background-color: rgb(45,45,55);
    color: rgb(255,255,255);
    padding: 5px;
    border: 1px solid rgb(60,60,70);
}</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
